apps/drec-ui/src/apps/certificate/view/pages/BlockchainInboxPage/BlockchainInboxPage.effects.ts

```diff
@@ -0,0 +1,62 @@
+import { useCertificateControllerGetAll } from '@energyweb/origin-drec-api-client';
+import { ListAction } from '@energyweb/origin-ui-core';
+import { useUser } from '../../../../../api';
+import { useAllDeviceFuelTypes, useApiAllDevices } from '../../../../device';
+import { useBlockchainInboxLogic } from '../../../logic';
+import { useBlockchainInboxPermissionsLogic } from '../../../logic/blockchain-inbox/permissions';
```

Comment:
you can use `api` & `apps/certificate/logic` since we are using import path alias.